python_modules/libraries/dagster-dbt/dagster_dbt/cli/app.py

```diff
@@ -85,11 +85,14 @@ def copy_scaffold(
 
     dbt_profiles_path = dbt_project_dir.joinpath(DBT_PROFILES_YML_NAME)
     dbt_profiles_yaml: Dict[str, Any] = yaml.safe_load(dbt_profiles_path.read_bytes())
```

Comment:
Could just pop the key instead.

```suggestion
    dbt_profiles_yaml: Dict[str, Any] = yaml.safe_load(dbt_profiles_path.read_bytes())
    
    # Remove the global configs when iterating profiles
    dbt_profiles_yaml.pop("config", None) 
```

Reply:
oh that's so much simpler, let do that.